modules/web/src/com/haulmont/cuba/web/app/ui/core/settings/messages.properties

```diff
@@ -24,11 +24,12 @@ modeSingle=Single Window
 
 changePassw=Change Password
 
+resetSettings=Reset
 resetScreensSettings=Reset settings of screens
 resetScreensSettings.description=Resets settings of all UI components in screens. For instance, settings of tables, \
- filters etc.
+ default filters etc.\nAre you sure you want to reset screens settings?
 resetScreensSettings.notification=All screens settings are resetted
-resetScreensSettings.warningNotification=Please, close other screes and try again
+resetScreensSettings.warningNotification=Please, close other screens and try again
```

Comment:
I'm not sure that it is a viable way to ask users about it. Let's check if we can perform force-logout without saving screen settings